flutter-idea/third_party/vmServiceDrivers/org/dartlang/vm/service/VmService.java

```diff
@@ -83,7 +83,7 @@ public class VmService extends VmServiceBase {
   /**
```

Comment:
The remaining files were all generated from a script, right? I'm not checking them.

Reply:
correct